### DISCUSSION TITLE
Can't Figure Out How to Get to Lustia

### DISCUSSION THREAD

**Discussion by duststar860** [ErinaSugino/Starbound-SBR-Mod#88] [Trouble Shooting] [answered]
Hello there;
I've been using these mods for a while (Sexbound and related mods) and when I found out that Lustbound had been added to Sexbound Reborn, I tried to craft the vending machine that gives you the Lustia ticket.
The vending machine thing is supposed to be crafted in the (C) crafting menu, right? But for some odd reason, it seems not to be there? Or at least, I've seen people calling it a "presentoir" and I can't seem to craft anything with that name.
I can craft the Alluring Toys Co Vending Machine, but it doesn't include the ticket either. I'm really confused, and at this point, I'm willing to just /spawnitem but I also can't find the item ID to do that! 
Any help, even just the "presentoir" ID or the Ticket ID would be great... Thank you, this mod is great <3

I hope I added the Log file correctly...
[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/14159845/starbound.log)

**Comment by ErinaSugino** [comment, marked as answer]
The Lustbound sextoy presentoir can be crafted at a Tier 1 workbench.

The IDs are `sexbound_sextoypresentoir` and `sexbound_onewaytickettolustia`.

**Comment by duststar860** [comment]
Thank you! I see where I went wrong, on the original Lustbound page, it said:
 "Where do I get access to the content? ~

Opening up your default crafting menu (C), you can make an object called the Sex Toy Presentoir, which will give you access to a handful of utility items as well as the mod's main star; the Sexboxes. "

But on the Loverslab SBR page it didn't seem to mention anywhere how to make it. I could've sworn I had already seen the Presentoir in a crafting menu too, so that explains a lot!

Edit: I just realized a huge mistake on my end, I have SBR 1.0 not 1.1, so I actually *don't* have access to Lustbound stuff. I'm gonna go update my Loverslab mods.
